src/main/java/ua/danit/final_project/repositories/ShiftCommentRepository.java

```diff
@@ -1,9 +1,23 @@
 package ua.danit.final_project.repositories;
 
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 import ua.danit.final_project.entities.ShiftComment;
+import ua.danit.final_project.entities.WorkShift;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'ua.danit.final_project.entities.WorkShift'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=85&issues=AWWG_p7_X2sY87p04Y77&open=AWWG_p7_X2sY87p04Y77)

[ISSUE_KEY]: AWWG_p7_X2sY87p04Y77